core/src/ast_builder/expr/function.rs

```diff
@@ -145,6 +153,22 @@ impl TryFrom<FunctionNode> for Function {
                 let start = start.try_into()?;
                 Ok(Function::Substr { expr, start, count })
             }
+            FunctionNode::Ltrim {
+                expr_node,
+                chars_node,
+            } => {
+                let chars = chars_node.map(TryInto::try_into).transpose()?;
```

Comment:
While reading through the following file, I found the repetition of the following line of code.
I thought about making the `Optional<Exprnode>` as our new node. 
I would like to ask for your ideas @panarch @ever0de